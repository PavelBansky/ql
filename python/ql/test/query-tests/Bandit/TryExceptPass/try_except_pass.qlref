Bandit/try_except_pass.ql