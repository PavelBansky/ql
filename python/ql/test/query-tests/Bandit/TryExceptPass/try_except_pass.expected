| Test.py:5:5:5:8 | Pass | Try, Except, Pass detected. |
| Test.py:12:5:12:8 | Pass | Try, Except, Pass detected. |
| Test.py:19:5:19:8 | Pass | Try, Except, Pass detected. |
| Test.py:35:5:35:8 | Pass | Try, Except, Pass detected. |
